drivers/usb/device/usb_dc_stm32.c

```diff
@@ -277,6 +298,9 @@ static int usb_dc_stm32_clock_enable(void)
 	 */
 #if defined(CONFIG_SOC_SERIES_STM32H7X)
 	LL_AHB1_GRP1_DisableClockSleep(LL_AHB1_GRP1_PERIPH_USB1OTGHSULPI);
+#elif defined(CONFIG_SOC_SERIES_STM32U5X)
+	LL_AHB2_GRP1_DisableClockStopSleep(LL_AHB2_GRP1_PERIPH_OTG_HS ||
```

Comment:
Disabling `LL_AHB2_GRP1_PERIPH_OTG_HS` is not stricly in line with this section covered by comment: "Disable ULPI interface (for external high-speed PHY) clock in sleep/low-power mode".
I'd suggest to split and add a specific comment.

Reply:
put in another commit  3689885f5ec4b5cc8f2db3d4916b8787bb9bd130 and

Reply:
LL_AHB2_GRP1_DisableClockStopSleep must disable sleep of the OTG AND PHY  clock  at the same time, else the usb connection is not detected